fvm/crypto/crypto.go

```diff
@@ -172,6 +173,43 @@ func verifySignatureFromRuntime(
 	return valid, nil
 }
 
+// TODO change int to fixed size int
+func ConstructNewPublicKey(publicKey *runtime.PublicKey,
```

Comment:
It would be clearer if we add "account" since there are `PublicKey` struct on different levels. 
```suggestion
func NewAccountPublicKey(publicKey *runtime.PublicKey,
```

Reply:
renamed at 2a97333